ddtrace/contrib/internal/bottle/trace.py

```diff
@@ -42,24 +37,21 @@ def wrapped(*args, **kwargs):
 
```

Comment:
This might be beyond the scope of the PR but ideally no integration should directly reference spans, tracers, or product specific configurations (ex: distributed_tracing). These details should be abstracted away (I guess this module is named `trace` so maybe this doesn't apply here)

Reply:
Yes agreed.